### PR TITLE
Sparklyr

### PR DESCRIPTION
**Pull request by robertwhiffin**
This brings the sparklyr bindings in, building on the auto-sql registration

### CODE REVIEW

**Comment by robertwhiffin** [comment]
snapshot removed from grep @milos-colic

**Review by milos-colic — APPROVED** [review]

**Review by milos-colic — APPROVED** [review]

**Comment by milos-colic** [comment]
@robertwhiffin this looks good to me.
shall we merge?

File: R/sparklyr-mosaic/enableMosaic.R [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#' @param indexSystem character, default="H3"
+#' @name enableMosaic
+#' @rdname enableMosaic
+#' @return None
+#' @export enableMosaic
+#' @examples
+#' \dontrun{
+#' enableMosaic()
+#' enableMosaic("ESRI", "H3")
+#' enableMosaic("ESRI", "BNG") # Not yet supported}
```
Comment (edurdevic) [inline R/sparklyr-mosaic/enableMosaic.R:16]: This is now supported

File: R/sparklyr-mosaic/sparklyrMosaic/DESCRIPTION [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Package: sparklyrMosaic
+Title: sparklyr bindings for Databricks Mosaic
+Version: 0.2.2
```
Comment (edurdevic) [inline R/sparklyr-mosaic/sparklyrMosaic/DESCRIPTION:3]: Version now is 0.3.3

**Review by edurdevic — APPROVED** [review]
LGTM

**Pull request merged**